pom.xml

```diff
@@ -20,7 +20,7 @@
 
     <properties>
         <java.version>17</java.version>
-        <spring.version>3.2.1</spring.version>
+        <spring.version>3.0.5</spring.version>
```

Comment:
What is the purpose of decreasing spring version?

Reply:
There was a problem in dependencies with Jackson Databind versioning.